testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6295,6 +6295,20 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     f.checkNull("log(10, cast(null as real))");
   }
 
+  @Test void testLogFuncByOneParameter() {
```

Comment:
can you javadoc this method? Why not add these to `testLogFunc`?

Reply:
add testLogfunc